src/models/model_loader.cpp

```diff
@@ -1,5 +1,7 @@
 #include "ed/models/model_loader.h"
 
+#include <boost/make_shared.hpp>
```

Comment:
Why is this added? Was something broken? Please elaborate

Reply:
I experienced this being a problem on my computer. I have a more changes on my computer which remove some boost dependencies where it wasn't needed. That showed the missing include here. Use here: https://github.com/tue-robotics/ed/blob/ce221ae162e32930f435e68412b59b572a84ae85/src/models/model_loader.cpp#L545